apps/prairielearn/src/ee/institution/utils.ts

```diff
@@ -1,38 +1,49 @@
-import {
-  loadSqlEquiv,
-  queryAsync,
-  queryOneRowAsync,
-  queryZeroOrOneRowAsync,
-} from '@prairielearn/postgres';
+import { loadSqlEquiv, queryOptionalRow, queryRow, queryRows } from '@prairielearn/postgres';
 
 import { config } from '../../lib/config';
+import {
+  AuthnProviderSchema,
+  InstitutionSchema,
+  SamlProviderSchema,
+  type AuthnProvider,
+  type Institution,
+  type SamlProvider,
+} from '../../lib/db-types';
 
 const sql = loadSqlEquiv(__filename);
 
-export async function getInstitution(institutionId) {
-  const institutionRes = await queryOneRowAsync(sql.select_institution, {
-    id: institutionId,
-  });
-  return institutionRes.rows[0];
+export async function getInstitution(institutionId: string): Promise<Institution> {
+  const institutionRes = await queryRow(
```

Comment:
Using singular-name-of-table for the variable name:
```suggestion
  const institution = await queryRow(
```

Reply:
Yep, good call.

Reply:
Done in 508d20f.